### ISSUE TITLE
New layout/functionality  for the simulation software run on the PC

### ISSUE THREAD

**Issue by fdapport** [FrSkyRC/ETHOS-Feedback-Community#4137]
Current simulation version : Ethos X20S Ver 1.5.8

This is not a issues  but a request of a new  feature to improve the use of the radio simulation  software. 

It will be very useful  for learning proposal to have a complete image of the current radio, not only the LCD but also the layout of the radio, include sticks, switches, knobs, etc. as was the simulation software of Open TX. 
All the switches, sticks, etc need to  move with the mouse instead of a keyboard. I know that I can interact with the simulation software with defined key on the keyboard like “KEY A” for the Switch A but I cannot see the current positions of the switch. This is not useful. 

This request is to share this idea without know the long term develop strategy for the creation, modification of the models on the radio directly on the PC of Ethos , as was in open TX.
Any feedback will be appreciated.

![image](https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/42831968/ac536468-51db-45bf-9df8-15f819e1393e)

**Comment by pmullen503** [comment]
That would be a welcome addition.

**Comment by bsongis-frsky** [comment]
Right. This is our current work, on the online simulator (which will be embedded inside Ethos Suite)

**Comment by MikeAg03** [comment]
Is this functionality in work or planned?  As it stands, I'm not sure how much value the simulator adds as it's hard to visually see the keyboard control/interaction/switch states/etc...

**Comment by fdapport** [comment]
The value of this new functionality is to semplify the test the new firmware versions before to implement it in the radio.

**Comment by mawzthefinn** [comment]
Is this still an issue with the new Simulator in Suite 1.7.0?

**Comment by fdapport** [comment]
Yes, 1.7.0 simulator is a great features. Look like great improvement... nice job!!
